frontend/amundsen_application/static/js/pages/SearchPage/SearchFilter/ToggleFilter/index.spec.tsx

```diff
@@ -0,0 +1,90 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as React from 'react';
+import { shallow } from 'enzyme';
+
+import { ToggleSwitch } from 'components/ToggleSwitch';
+
+import globalState from 'fixtures/globalState';
+
+import { GlobalState } from 'ducks/rootReducer';
+
+import { ResourceType } from 'interfaces';
+
+import {
+  ToggleFilter,
+  ToggleFilterProps,
+  mapDispatchToProps,
+  mapStateToProps,
+} from '.';
+
+describe('ToggleFilter', () => {
+  const setup = (propOverrides?: Partial<ToggleFilterProps>) => {
+    const props: ToggleFilterProps = {
+      categoryId: 'pii',
+      filterName: 'PII',
+      helpText: 'Filters out all resources cotaining PII',
+      checked: true,
+      applyFilters: jest.fn(),
+      ...propOverrides,
+    };
+    // eslint-disable-next-line react/jsx-props-no-spreading
+    const wrapper = shallow<ToggleFilter>(<ToggleFilter {...props} />);
+    return {
+      props,
+      wrapper,
+    };
+  };
+
+  describe('render', () => {
+    it('creates ToggleSwitch', () => {
+      const { wrapper } = setup();
+      const expected = 1;
+      const actual = wrapper.find(ToggleSwitch).length;
+      expect(actual).toEqual(expected);
+    });
+  });
+
+  describe('handleChange', () => {
+    it('calls props.applyFilters with expected parameters', () => {
+      const { props, wrapper } = setup();
+      const applyFiltersSpy = jest.spyOn(props, 'applyFilters');
```

Comment:
Probaly this is the same as doing:
``` props.applyFilters.mockClear()```
Are you sure you need to clear the mock though?